api/user.go

```diff
@@ -218,6 +220,53 @@ func CreateUser(c *Context, team *model.Team, user *model.User) *model.User {
 	}
 }
 
+func fireAndForgetAddDirectChannels(user *model.User, team *model.Team) {
+	go func() {
+		AddDirectChannels(user.Id, team.Id)
+	}()
+}
+
+func AddDirectChannels(userId, teamId string) []*model.Preference {
```

Comment:
We don't want to show all users in the sidebar by default or else the list will become massive for anything other than small teams. We can just pick 10 users semi-randomly to fill out the list so that it is populated and not empty. 

This is actually redundant if you consider the lazy version that we do when the getPreferences api is called so I might just remove doing it here.